src/sections/Community/Handbook/community.js

```diff
@@ -172,35 +175,35 @@ const CommunityGuide = () => {
             </p>
             <ul style={badgeListStyle}>
               <li>
-                <img src={CommunityLogo} style={badgeStyle} />
+                <StaticImage src={CommunityLogo} style={badgeStyle} alt="badgeStyle"/>
```

Comment:
`alt` text over here should be `Community Logo`.
Please update similarly in other places as well

Reply:
:+1: